mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -912,9 +1042,12 @@
         </InfoArray>
     </CAbilMorph>
     <CAbilEffectTarget id="TimeWarp"/>
+    <CAbilTrain id="TrainQueen">
+        <InfoArray index="Train1" Effect=""/>
```

Comment:
Queens are no longer unable to receive certain orders for 0.6 seconds after spawning (0.8 seconds on normal speed)

The Queen received a behavior at birth from the training ability which disables move and attack abilities for 0.8 seconds on normal speed. This did not prevent other abilities. This was likely intended to have an uninterruptible spawning animation where the Queen plays her unburrow animation

Since players could perform some actions already, this was more of a hindrance than helpful. In most scenarios, the Queen will still play her animation at birth.
There could be some animation issues when she is birthed and immediately starts attacking an enemy, though